src/components/MainImage/MainImage.js

```diff
@@ -0,0 +1,19 @@
+import { Grid } from '@mui/material';
+import { mainImages } from '../Data/MainImagesData.js';
+import Box from '@mui/material/Box';
+import { styles } from '../styles';
+
+const MainImage = () => {
+  return (
+    <Grid>
+      {mainImages.map(mainImage => (
+        <Box key={mainImage.id}>
```

Comment:
Try to use https://react-slick.neostack.com/ to implement the slider.
To make it easier for you, place slide buttons under the image.
![image](https://user-images.githubusercontent.com/61732059/195200321-af183677-e996-4d83-8ecf-47e82da97330.png)